framework/src/meshgenerators/XYDelaunayGenerator.C

```diff
@@ -163,6 +173,22 @@ XYDelaunayGenerator::XYDelaunayGenerator(const InputParameters & parameters)
   for (auto hole_i : index_range(_stitch_holes))
     if (_stitch_holes[hole_i] && (hole_i >= _refine_holes.size() || _refine_holes[hole_i]))
       paramError("refine_holes", "Disable auto refine of any hole boundary to be stitched.");
+
+  // Copied from MultiApp.C
```

Comment:
Maybe create a method in `MooseUtils` so that both MultiApp.C and this class can call it.

Reply:
It's a good idea. MooseUtils is a little long already we could do a DelimitedFileReadingUtils
Wont require it